### PR TITLE
Include day and month abbreviations for Catalan

### PR DESCRIPTION
**Pull request by NestorTejero**
Sources:
- https://www.ub.edu/cub/llistat_ab.php?taula=DS&ordre=mots_t&idioma=1
- https://aplicacions.llengua.gencat.cat/llc/AppJava/index.html?action=Principal&method=detall&input_cercar=abreviatures+mesos&numPagina=1&database=FITXES_PUB&idFont=8402&idHit=8402&tipusFont=Fitxes+de+l%2527Optimot&numeroResultat=1&databases_avansada=&categories_avansada=&clickLink=detall&titol=abreviatures+dels+mesos+de+l%2527any&tematica=&tipusCerca=cerca.normes

## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [ ] 🧪 Added **tests** for changed code.
- [ ] 🛠️ All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [ ] 🧹 All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [ ] 📚 Updated **documentation** for changed code.
- [ ] ⏩ Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

### CODE REVIEW

File: arrow/locales.py [hunk 1]
```diff
@@ -2832,7 +2832,7 @@ class HindiLocale(Locale):
     month_abbreviations = [
         "",
         "जन",
-        "फ़र",
+        "फ़र",
```
Comment (NestorTejero) [inline arrow/locales.py:2835]: I edited the file through the GitHub page, so the browser may have refilled some characters

**Review by systemcatch — APPROVED** [review]
Thank you for improving the Catalan locale!

**Review by jadchaar — APPROVED** [review]
Looks good, but was the removal of title casing intentional @NestorTejero?

**Comment by jadchaar** [comment]
Seems like the title casing is in fact intentional: https://wordinfo.info/unit/3230. Should be good to merge this once CI passes.

**Comment by NestorTejero** [comment]
> Looks good, but was the removal of title casing intentional @NestorTejero?

yes! In Catalan, month and week day names are not capitalized (unless in the beginning of a sentence). I included the sources in [my first comment](https://github.com/arrow-py/arrow/pull/853#issue-492938310). Thanks for cross checking.

**Pull request merged**